src/tools/dart-run.md

```diff
@@ -129,13 +129,30 @@ $ dart run foo arg1 arg2
 ## Debugging
 
 To enable debugging, pass one or more debugging options.
-Here's an example of enabling [`assert` statements][assert]:
+
+To enable [`assert` statements][assert],
+specify the `--enable-asserts` flag:
 
 ```terminal
 $ dart run --enable-asserts tool/debug.dart
 ```
 
-See `dart run --help` for details.
+To enable debugging through [Dart DevTools](/tools/dart-devtools),
+a set of debugging and performance tools,
+specify the `--observe` flag.
+Optionally add `--pause-isolates-on-start`
+to break execution at the start of the specified script:
```

Comment:
**issue:** For compound sentences, lead with the _why_, then the _how_. Also, you don't need _optionally_. All of these commands are optional.

```suggestion
To break execution at the start of the specified script, add `--pause-isolates-on-start`.
```